### DISCUSSION TITLE
Backup only files with specific extension

### DISCUSSION THREAD

**Discussion by Ritzelschrubber** [outdoorbits/little-backup-box#153] [Q&A]
Is there a possibility/setting in LBB to backup only files with a specific extension (like ".raf") and ignore f.i. "jpg's"?

**Comment by outdoorbits** [comment]
Not yet. But because of the high number of file types, some cameras create and the universal use of little backup box, I wouldn't prefer that. Maybe an exclude list would do the job for you?

**Reply by Ritzelschrubber** [reply]
Thanks for the reply. Exclude would be perfect for me - is there a setting anywhere?

**Comment by outdoorbits** [comment]
I'm in holiday and back next week. Then I will check that. I'll reply.

**Reply by Ritzelschrubber** [reply]
Thanks - no need to hurry - enjoy your holiday

**Comment by outdoorbits** [comment]
To exclude files seems to be impossible for camera- or smartphone-backups, because gphoto2 does not support exclusions. There maybe possible workarounds (get file list first and download only files not excluded one by one). But I'm sure this would decrease backup-speed significantly.
Of course, excluding file-patterns work for backups from file-system (e.g. devices like USB-sticks). But this would mean to introduce a different behavior in both backup-types.
I'm not sure, what's the best way. What type of backup is relevant to you?

**Reply by Ritzelschrubber** [reply]
I guess I simply clean up the card and delete jpgs before starting backup. That's good enough for my purpose.. Thanks for your support

**Reply by outdoorbits** [reply]
And I guess this would be the less confusing way. ;-)
